src/main/java/Duke.java

```diff
@@ -6,5 +14,167 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("\t------------------------------------------");
+        System.out.println("\tHello there! I'm Duke.");
+        System.out.println("\tWhat can I help you with?");
+        System.out.println("\t__________________________________________\n");
+
+        ArrayList<Task> tasks = new ArrayList<>();
+        boolean runLoop = true;
+
+        while (runLoop) {
+            String userInput = sc.nextLine().trim();
+
+            if (userInput.equalsIgnoreCase("bye")) {
+                exitProgram();
+                runLoop = false;
+            } else if (userInput.equalsIgnoreCase("list")) {
+                listTasks(tasks);
+            } else if (userInput.split(" ")[0].equalsIgnoreCase("done")) {
+                markTask(userInput, tasks);
+            }
+            else if (userInput.split(" ")[0].equalsIgnoreCase("delete")) {
+                deleteTask(userInput, tasks);
+            }
+            else {
+                addTask(userInput, tasks);
+            }
+        }
+    }
+
+    public static void exitProgram() {
+        // exits the program
+        System.out.println("\t------------------------------------------");
```

Comment:
If you're using this string a lot, perhaps consider storing it as a constant variable that you could re-use elegantly.